datahandler/userHandle.py

```diff
@@ -176,6 +176,32 @@ def addTextMindCooldown(
         else:
             print(f"\tUser {userID} is on Cooldown. CurrentTime: {delta_t}")
 
+    def addReactionMindCooldown(
```

Comment:
This method shares a lot of code with `addTextMindCooldown`.
Extract the cooldown logic to a new function `checkUserCooldown`.

Reply:
New checkUserCooldown(...): --> bool
**done**

New addTextXPMindCooldown(...): --> addTextXP(...)
if is reaction on bot or text message --> set is_text/is_reaction
if is_text --> increase TextCount
if b_checkUserCooldown --> 
    if is_text --> get Textcooldown from config --> set cooldown
    if is_reaction --> set cooldown to 10 ??? _maybe add ReactionCooldown to config_
    if  checkUserCooldown(...) --> next or 'return and print'
addTextXP(...)